packages/select/src/components/query-list/queryList.tsx

```diff
@@ -443,6 +498,19 @@ function pxToNumber(value: string | null) {
     return value == null ? 0 : parseInt(value.slice(0, -2), 10);
 }
 
+function getMatchingItem<T>(query: string, { items, itemEqualsQuery }: IQueryListProps<T>): T | undefined {
+    if (Utils.isFunction(itemEqualsQuery)) {
+        // .find() doesn't exist in ES5. Alternative: use a for loop instead of
```

Comment:
AFAIK, `Array.prototype.find` should be supported in blueprint's environment.. can you elaborate here why that doesn't work?

Reply:
@adidahiya Yep, the compile step was failing on Circle. [Here is the failing build](https://circleci.com/gh/palantir/blueprint/24768?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) from a few commits back.

Reply:
that's unfortunate, I wish we had added it to the environment requirements. oh well.